projects/EWMS_Riedel_Startup.yaml

```diff
@@ -0,0 +1,8 @@
+Department: Physics
+Description: To develop an Observation Management System Framework which will help alleviating multiple types of astrophysical data and respective workflow management. The Event Workflow Management System (EWMS)is a workload manager designed for processing events (simulated readouts from a particle physics detector, recorded data points, images, etc.) with complex workflows.
+FieldOfScience: Astrophysics
+Organization: Event Workflow Management System (EWMS)
```

Comment:
Is EWMS really an organization? Shouldn't it be "University of Wisconsin-Madison" or something?

Reply:
A google search of the grant shows that it went to Madison, so I think you're correct.